### DISCUSSION TITLE
New Script Idea for Matrix HomeServer & Web-Element, Zammad, Kimai, Zabbix, Bookstack, Gitea

### DISCUSSION THREAD

**Discussion by florianwilde** [tteck/Proxmox#2307] [Ideas]
Hello,
are there any suggestions for the following services:
- Matrix & Element HomeServer 
- Kimai time recording
- Zammad ticket system
- bookstack
- gitea
- Zabbix

**Comment by LoadingStill** [comment]
+1 for Gitea

**Reply by tteck** [reply]
Use the TurnKey LXC Appliances script, it has Gitea.

**Reply by LoadingStill** [reply]
Thank you.  I was unaware of that.

**Reply by sadekd** [reply]
Zabbix ❤️

**Reply by Deadfuntik** [reply]
Matrix 👍

**Reply by schattenphoenix** [reply]
I think a synapse / matrix server might be really cool

**Reply by Earthlingz** [reply]
hoping for zammad script

**Reply by sosher** [reply]
Kimai would be great

**Reply by halfebyte** [reply]
Greatest praise!
Zammad, that would make me very happy.

**Comment by nodeam** [comment]
Matrix would be nice.

**Comment by jsapede** [comment]
difinitely matrix

**Comment by 19Fidel84** [comment]
oh yea Matrix Server Lxc whe nice

**Comment by miregal89** [comment]
for zammad, zabbix, matrix you can use this script

https://github.com/bashclub/zamba-lxc-toolbox

For Tutorial in Youtube Search Zamba Toolbox

**Reply by tteck** [reply]
Be cautious and thoroughly evaluate scripts and automation tasks obtained from external sources.
